src/homepage/index.html

```diff
@@ -72,8 +73,8 @@ <h2>About</h2>
               <p class="align-left">Create your <b>own personal solution</b> of emails treatment by assembling the components you need thanks to the Inversion of Control mail platform offered and  go further customizing filtering and routing rules using <b>James Mailet Container</b>.</p>
               <div class="about-table">
                 <h3 class="tb-h3">James from a technical point of view</h3>
-                <b>Complete portability</b> (100% pure Java)<br/>
-                <b>Built with Java 8</b> and running on the <b>JRE 1.8</b>.<br/><br/>
+                <b>Complete portability</b> (100% pure JVM: Java &amp; Scala)<br/>
+                <b>Built with Java 11</b> and running on the <b>JRE 1.8</b>.<br/><br/>
```

Comment:
we can still build it with java 8, I would not advertise that.

Reply:
wrong: if you build it with java8 you get the --release flags error

Reply:
> wrong: if you build it with java8 you get the --release flags error

I know that but it's a very small detail as you could technically remove the use of this flag an it build with JDK8.

On the other side, we forbid the use of any JDK11 API.

What is the closest description of the reality?

Reply:
currently users are complaining they cannot build James with JDK8. The solution we give them is to use JDK11.

If you have a simple patch to allow James to build on JDK8 I will be happy to integrate it and change my wording.

Reply:
Just to be sure, to the question

> What is the closest description of the reality?

You think "JDK11" is the best answer.

If so, let's opt for that